src/app/zap-templates/zcl/data-model/chip/drlc-cluster.xml

```diff
@@ -144,37 +118,34 @@ limitations under the License.
        really make sense in all the places where the struct is used, and the spec
        does not really define other things in a way that would work with that at
        the moment. -->
-  <struct name="LoadControlEventStruct">
-    <cluster code="0x0096"/>
-    <item fieldId="0" name="EventID" type="octet_string" length="16"/>
-    <item fieldId="1" name="ProgramID" type="octet_string" length="16" isNullable="true"/>
-    <item fieldId="2" name="Control" type="EventControlBitmap" />
-    <item fieldId="3" name="DeviceClass" type="DeviceClassBitmap" />
-    <item fieldId="4" name="EnrollmentGroup" type="int8u" optional="true" />
-    <item fieldId="5" name="Criticality" type="CriticalityLevelEnum" defaut="0" />
-    <item fieldId="6" name="StartTime" type="epoch_s" isNullable="true" />
-    <item fieldId="7" name="Transitions" array="true" type="LoadControlEventTransitionStruct" />
+  <struct name="LoadControlEventStruct" isFabricScoped="true">
+    <cluster code="0x0096"/>
+    <item fieldId="0" name="EventID" type="octet_string" length="16" default="0" minLength="16"/>
+    <item fieldId="1" name="ProgramID" type="octet_string" length="16" isNullable="true" isFabricSensitive="true" minLength="16"/>
+    <item fieldId="2" name="Status" type="LoadControlEventStatusEnum" optional="true" isFabricSensitive="true" min="0x00" max="0x0C"/>
+    <item fieldId="3" name="Control" type="EventControlBitmap" isFabricSensitive="true" default="0x0001" min="0x0000" max="0x0001"/>
+    <item fieldId="4" name="Criticality" type="CriticalityLevelEnum" isFabricSensitive="true" default="0x00" min="0x00" max="0x06"/>
+    <item fieldId="5" name="StartTime" type="epoch_s" defaut="0" isNullable="true" isFabricSensitive="true"/>
+    <item fieldId="6" name="Transitions" type="LoadControlEventTransitionStruct" array="true" isFabricSensitive="true"/>
   </struct>
 
-  <struct name="LoadControlProgramStruct">
+  <struct name="LoadControlProgramStruct" isFabricScoped="true">
     <cluster code="0x0096"/>
-    <item fieldId="0" name="ProgramID" type="octet_string" length="16"/>
-    <item fieldId="1" name="Name" type="long_char_string" length="32"/>
-    <item fieldId="2" name="EnrollmentGroup" type="int8u" isNullable="true" min="1" max="0xFF"/>
-    <item fieldId="3" name="RandomStartMinutes" type="int8u" isNullable="true" min="0x00" max="0x3C"/>
-    <item fieldId="4" name="RandomDurationMinutes" type="int8u" isNullable="true" min="0x00" max="0x3C"/>
+    <item fieldId="0" name="ProgramID" type="octet_string" length="16" default="0" minLength="16"/>
+    <item fieldId="1" name="Name" type="char_string" length="32"/>
+    <item fieldId="2" name="RandomStartMinutes" type="int8u" isNullable="true" max="60"/>
+    <item fieldId="3" name="RandomDurationMinutes" type="int8u" isNullable="true" max="0x3C"/>
```

Comment:
It's interesting that alchemy output 60 for one of those and 0x3C for the other.  Maybe check with @hasty what's going on there?

Reply:
Not sure about this - will ask look into it see if I can explain - otherwise ask Hasty - Thanks for the review

Reply:
This is a side effect of alchemy trying to be gentle with existing XML, and leaving formatting alone when the values are equivalent, since people used random strategies for hex vs int. 

In this case, field #&#8203;2 was removed, so field #&#8203;3 became field #&#8203;2, field #&#8203;4 became field #&#8203;3, etc. When it attempted to update field #&#8203;2 to match what used to be field #&#8203;3, the max value of "0xFF" is not equivalent to "60", so it re-rendered the "max" attribute, which defaults to using base 10. But when it got to field #&#8203;3, which used to be field #&#8203;4, it already had a max attribute set to "0x3C", which is equivalent to "60", so it left it alone.

If you update it to "60", alchemy will leave it alone on future runs. Whenever the ZAP XML as a whole is updated to have predicatable formatting for numbers, we could get rid of this gentleness.